mantidproject/mantid
# Add LOQ system tests for SANS legacy and TOML files

**Description of work.**

Adds system tests to check the same result is produced from some
representative files provided by LOQ instrument scientists, against the
new TOML files that is replacing them

**To test:**
- Ensure new tests run and pass
- Proof read docs to check for silly typos

There is no associated issue. - This is part of the ongoing effort to convert a few representative TOML files for the SANS group





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
